### DISCUSSION TITLE
Add 'Convert to issue' option to Tasklist when shown inside the sidebar

### DISCUSSION THREAD

**Discussion by MakePixelsWork** [community/community#43327] [Projects and Issues]
### Select Topic Area

Product Feedback

### Body

When you preview an Issue inside your Project, you get to see the info inside a Sidebar. When you create a Tasklist inside that, the options to Convert those to issues is not available.

Example:
![image](https://user-images.githubusercontent.com/56885246/210625536-b83a1136-4126-44c1-b4f8-1e6a57148c5b.png)

Can this be added? This saves the user time to convert their tasks to issues, without having to leave this Sidebar window.

**Comment by hawkkiller** [comment]
I agree, it would be really great feature that saves our time and improves UX.
